robottelo/ui/locators.py

```diff
@@ -957,7 +957,8 @@ def __iter__(self):
     "host.vm_memory": (By.ID, "host_compute_attributes_memory"),
     "host.network_type": (
         By.ID, "host_compute_attributes_nics_attributes_0_type"),
-    "host.network": (By.ID, "host_compute_attributes_nics_attributes_0_type"),
+    "host.network": (
+        By.ID, "host_compute_attributes_nics_attributes_0_bridge"),
```

Comment:
Did this change happen recently in 6.1.6 or upstream? I just wanted to make sure upstream/downstream are intact.